doc/11-Concepts.md

```diff
@@ -0,0 +1,77 @@
+
+### Visual
+Each element has a so-called visual. It's purpose is to highlight certain elements in long lists and give an intuitive overview about the underlying objects' condition. 
+In the case of host or service lists, for example, the state is displayed. This makes it immediately clear if and where there are problems that need to be taken care of, immediately from the overview.
+
+### Title
+The title briefly describes the state of the list element as an addition to the visual. For example, it contains the information that a host is currently DOWN. While the visual gives an intuitive impression, the title explains what exactly happened.
+
+### Meta
+The meta area is for additional information, usually displays of time. In the Host and Service Lists, you can see how long the object has been in the corresponding state.
+
+### Caption
+The caption area contains detailed information about the list item. In the case of hosts and services this would be the plugin output. For comments and downtimes the comment texts of the user are displayed here. To make the default list view compact and consistent, long texts are truncated to a maximum of two lines.
+
+## Different detail degrees
+
+The level of detail of the host and service lists is now selectable - the above image shows the default list item, but there are two more iterations:
+
+In the compact mode the entire list item is displayed in one single line.
+Given, that there is enough space, the caption is placed in the same line as the rest and trimmed to fit.
+![CompactAnatomy](res/ListItemAnatomyCompact.jpg "A more compact list item that fits in one line")
+
+The detailed mode displays the entire caption, which makes it possible to react to problems directly from the list view.
+![DetailedAnatomy](res/ListItemAnatomyDetailed.jpg "A more detailed list item that spans four lines")
+
+## Overdue Items
+
+In host and service lists overdue checks are now highlighted. This makes it immediately apparent which objects may no longer be up-to-date. The red badge contains information about how long the check has overdue.
+![OverdueListAnatomy](res/ListAnatomyOverdue.jpg "A list with a highlighted item")
+
+## Downtimes
+Downtime lists have an extra information to be displayed: the progress.
+How far along the downtime is, is now immeadiately visible via the progress bar at the top of each list item.
+In the only square visual, we display how much time is left until the downtime ends.
+![DowntimeListItem](res/ListItemDowntimeAnatomy.jpg "A downtime list item with the added progress bar in the top")
+
+
+# Modal
+If you wanted to create a comment or set a downtime from the detail area, the dialog was displayed in another column, which meant that the contents of the left column were lost. In the icingadb module, there is now a modal element for short interaction dialogues. For actions in the detail area, a modal dialog will now appear. This preserves the left list column and thus the context better.
```

Comment:
Remove the first sentence as it compares to something unmentioned. 

Start with 

```
The modal element introduces short interaction dialogues. For actions in the detail area, a modal dialog will appear. This preserves the left list column and thus the content better.
```

Also, avoid `now` again.